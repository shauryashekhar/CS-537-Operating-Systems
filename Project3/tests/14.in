sleep 1 &
wait 0
sleep 1 &
wait 1

