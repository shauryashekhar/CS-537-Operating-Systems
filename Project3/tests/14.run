./mysh tests/14.in
