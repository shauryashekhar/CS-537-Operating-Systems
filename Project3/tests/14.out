sleep 1 &
wait 0
JID 0 terminated
sleep 1 &
wait 1
JID 1 terminated

